cf/models/user_roles.go

```diff
@@ -27,3 +27,15 @@ var SpaceRoleToUserInput = map[string]string{
 	SPACE_DEVELOPER: "SpaceDeveloper",
 	SPACE_AUDITOR:   "SpaceAuditor",
 }
+
+var ApiToOrgRole = map[string]string{
+	"manager_guid":         ORG_MANAGER,
+	"billing_manager_guid": BILLING_MANAGER,
+	"auditor_guid":         ORG_AUDITOR,
+}
+
+var ApiToSpaceRole = map[string]string{
+	"managed_spaces": SPACE_MANAGER,
+	"developer_guid": SPACE_DEVELOPER,
+	"audited_spaces": SPACE_AUDITOR,
+}
```

Comment:
Suggestion -
We can rename ApiToOrgRole  as QueryParmToOrgRole and ApiToSpaceRole  with QueryParmToSpaceRole